packages/liveblocks-core/src/room.ts

```diff
@@ -126,6 +126,7 @@ type RoomEventCallbackMap<
   error: Callback<Error>;
   connection: Callback<ConnectionStatus>;
   history: Callback<HistoryEvent>;
+  storage: Callback<StorageUpdate[]>;
```

Comment:
I had a typescript error because this was missing, but it's not related to my changes I don't think.

Reply:
For full context, the `.subscribe(...)` API is complicated because it has so many overloads. It has been historically really hard to type out in TypeScript.

Last year, I refactored things internally to have individual event sources for each event, so subscribing to specific events is much easier, because each event source has a specific payload type. Then, I reimplemented `.subscribe()` on top of those, as a backward-compatible layer with the API as it has always existed (aka today's API with many overloads).

These APIs also exist, and are recommended for internal use:

```ts
room.events.connection.subscribe(callback);  // .subscribe('connection', callback) calls this internally
room.events.me.subscribe(callback);          // .subscribe('my-presence') calls this internally
room.events.storage.subscribe(callback);     // .subscribe(root, callback) calls this internally
...
```

We have not advertised this API publicly (yet), but we could if we wanted to. It's stable, and here to stay, and I think slightly nicer to use. We just didn't want to cause confusion by having multiple subscription APIs. The single kitchen-sink style `room.subscribe()` API is just a wrapper around those more type-safe and cleaner APIs, and should only be used externally (as a Liveblocks user).

Reply:
removed the dead code!